networking.go

```diff
@@ -160,16 +162,33 @@ func startSSFUDP(s *Server, addr *net.UDPAddr, tracePool *sync.Pool) {
 	}()
 }
 
-func startSSFUnix(s *Server, addr *net.UnixAddr) {
+func startSSFUnix(s *Server, addr *net.UnixAddr) <-chan struct{} {
```

Comment:
Can we document (function documentation is fine) what the return channel here represents?

Reply:
Added!